ecf/scripts/workflow_manager/cycled/workflow_monitor.ecf

```diff
@@ -0,0 +1,119 @@
+#PBS -S /bin/bash
+#PBS -N %RUN%_workflow_monitor_%CYC%
+#PBS -j oe
+#PBS -q %QUEUE%
+#PBS -A %PROJ%-%PROJENVIR%
+#PBS -l walltime=06:00:00
+#PBS -l place=vscatter,select=1:ncpus=1:mem=1GB
+#PBS -l debug=true
+
+model=gfs
+%include <head.h>
+%include <envir-p1.h>
+
+set -x
+
+export NET=%NET:gfs%
+export RUN=%RUN%
+export CDUMP=%RUN%
+
+############################################################
+# Load modules
+############################################################
+module load PrgEnv-intel/${PrgEnv_intel_ver}
+module load craype/${craype_ver}
+module load intel/${intel_ver}
+module load cray-mpich/${cray_mpich_ver}
+module load cray-pals/${cray_pals_ver}
+module load python/${python_ver}
+
+module list
+
+export edate=%EDATE%
+export cyc=%CYC%
+export cycle=t${cyc}z
+
+GDATE=$($NDATE -06 $CDATE)
```

Comment:
Every thing from below this point does not belong in a job card.
A job card is not to be treated like a script.
Its purpose is to call a J-Job.  It seems this needs a J-Job similar to post manager.

Reply:
Will create a new JJob as requested.

Reply:
@arunchawla-NOAA and @aerorahul, do I need to create a new JJob or can I retain my current solution? I need your guidance.

Reply:
I would like this in a job card please to conform with existing norms and practices.

Reply:
Ok. Will create a new job card.

Reply:
A new JJOB has been created.